include/xsimd/config/xsimd_config.hpp

```diff
@@ -251,7 +251,7 @@
  *
```

Comment:
I think the problem it located above, so when "#ifdef __ARM_NEON" is not enough.

Reply:
oh, I missed that indeed

Reply:
@ricotz maybe better like this?

Reply:
I guess the problem is on our side and needs to be handled downstream.
So if there is no actual problem then please hold back those changes.

Reply:
ack, please ping me whenever you find an out.

Reply:
Thank for looking into it, but this is change is not reasonable for upstream.
It is fixed downstream by respecting specifics of Ubuntu's toolchain.